src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py

```diff
@@ -77,6 +77,36 @@
     short-summary: commands to configure hostnames
 """
 
+helps['appservice web config ssl'] = """
+    type: group
+    short-summary: commands to configure ssl certs
```

Comment:
That doesn't make English sense. As an acronym it should be capitalized in the short summary. ?